### PR TITLE
Mention @ in the placeholder message

### PR DESCRIPTION
**Pull request by mkondratek**
## Test plan
1. runIde
2. see Cody tool window

|   |   |
| --- | --- |
| image | image |

### CODE REVIEW

**Comment by taylorsperry** [comment]
Thanks, @mkondratek ! I'll defer to @danielmarquespt on design, but I think this is already an improvement. Is it possible to suggest the open file as soon as I type `@` (like VS Code does)? I feel like you type `@` and expect something to happen / don't know what to do next. 
Screenshot 2024-02-08 at 4 47 34 PM

**Comment by mkondratek** [comment]
Taylor, it's fixed by #555 🚀

File: src/main/resources/CodyBundle.properties [hunk 1]
```diff
@@ -108,3 +108,5 @@ popup.select-chat=Select Chat
 popup.remove-chat=Remove Chat
 popup.remove-all-chats=Remove All Chats
 
+PromptPanel.ask-cody=Message (use @ to include files)...
```
Comment (pkukielka) [inline src/main/resources/CodyBundle.properties]: @danielmarquespt WDYT?

To me that `"Message"` alone looks very incomplete.
I would prefer something like fix of old and new:
`"Ask a question about this code (you can use @ to include files)..."`
Unless that is too wordy?

Comment (mkondratek) [reply src/main/resources/CodyBundle.properties]: I just tried to make it as similar to VS Code as possible

Comment (mkondratek) [reply src/main/resources/CodyBundle.properties]: VS Code: 
image

Comment (danielmarquespt) [reply src/main/resources/CodyBundle.properties]: Let's go with:

`Message (type @ to include specific files as context)`

if it is a follow-up, (low priority)

`Follow-up message (type @ to include specific files as context)`

**Comment by mkondratek** [comment]
I applied the fixes:

|.  |.  |
| --- | --- |
| image | image |

**Review by Sa1to — APPROVED** [review]
LGTM
